requirements.txt

```diff
@@ -1,4 +1,4 @@
 pytest==2.9.2
 pytest-selenium
 pytest-xdist==1.15.0
-requests==2.11.0
```

Comment:
I think it is ok, since it passes all the tests and requests has released the version 2.11.1